sysid-application/src/main/native/cpp/analysis/AnalysisManager.cpp

```diff
@@ -598,6 +598,7 @@ AnalysisManager::AnalysisManager(std::string_view path, Settings& settings,
   // Get the rotation -> output units factor from the JSON.
   m_unit = m_json.at("units").get<std::string>();
   m_factor = m_json.at("unitsPerRotation").get<double>();
+  WPI_DEBUG(m_logger, "Parsing units per rotation as {}, {}", m_unit, m_factor);
```

Comment:
```suggestion
  WPI_DEBUG(m_logger, "Parsing units per rotation as {} {} per rotation", m_factor, m_units);
```
This will make it print `Parsing units per rotation as 0.2 Meters per rotation`.